packages/graphics/src/Graphics.js

```diff
@@ -262,10 +263,11 @@ export default class Graphics extends Container
      * @param {PIXI.Matrix} [matrix=null] Texture matrix to transform texture
      * @param {number} [alignment=0.5] - alignment of the line to draw, (0 = inner, 0.5 = middle, 1 = outter)
      * @param {boolean} [native=false] - If true the lines will be draw using LINES instead of TRIANGLE_STRIP
+     * @param {string} [lineJoin='miter'] - shape used to join two line segments where they meet
      * @return {PIXI.Graphics} This Graphics object. Good for chaining method calls
      */
     lineTextureStyle(width = 0, texture = Texture.WHITE, color = 0xFFFFFF, alpha = 1,
-        matrix = null, alignment = 0.5, native = false)
+        matrix = null, alignment = 0.5, native = false, lineJoin = LINE_JOIN.MITER)
```

Comment:
Argument and doc should probably be added to `lineStyle` too. I think I will refactor these argument into an object in a future PR. This method signature is getting too crazy.